src/sentry/mail/adapter.py

```diff
@@ -63,7 +63,12 @@ def rule_notify(
         project = event.group.project
         extra["project_id"] = project.id
 
-        if digests.enabled(project):
+        # XXX(workflow): remove this after wf2.0 experiment over
+        if target_type == ActionTargetType.RELEASE_MEMBERS:
+            notification = Notification(event=event, rules=rules)
+            self.notify(notification, target_type, target_identifier)
+
+        elif digests.enabled(project):
```

Comment:
Actually, wouldn't it be simpler to just do 
`digests.enabled(project) and target_type != ActionTargetType.RELEASE_MEMBERS`?